src/Common/src/TypeSystem/IL/Stubs/PInvokeILEmitter.cs

```diff
@@ -60,37 +59,25 @@ private static Marshaller[] InitializeMarshallers(PInvokeMethodData pInvokeMetho
 
         private MethodIL EmitIL()
         {
-            // We have 4 code streams:
-            // - _marshallingCodeStream is used to convert each argument into a native type and 
-            // store that into the local
-            // - callsiteSetupCodeStream is used to used to load each previously generated local
-            // and call the actual target native method.
-            // - _returnValueMarshallingCodeStream is used to convert the native return value 
-            // to managed one.
-            // - _unmarshallingCodestream is used to propagate [out] native arguments values to 
-            // managed ones.
```

Comment:
I like having those comments. Especially once you have cleanup/eception cleanup streams those can become confusing. Having a simple code here illustrates the purpose of each stream would be very nice.